microprofile-lra/README.adoc

```diff
@@ -0,0 +1,592 @@
+endpoints:
+
+- `GET /participant1/work` - work action of Participant 1
+- `GET /participant2/work` - work action of Participant 2
+- `PUT /participant1/compensate` - compensating action of Participant 1
+- `PUT /participant2/compensate` - compensating action of Participant 2
+- `PUT /participant1/complete` - complete action of Participant 1
+- `PUT /participant2/complete` - complete action of Participant 2
+
+
+// System Requirements
+include::../shared-doc/system-requirements.adoc[leveloffset=+1]
+
+// Use of {jbossHomeName}
+include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
+
+// Back Up the {productName} Standalone Server Configuration
+include::../shared-doc/back-up-server-standalone-configuration.adoc[leveloffset=+1]
+
+// Start the {productName} Standalone Server
+include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+1]
+
+[[configure_the_server]]
+== Configure the Server
+
+You can configure the LRA extensions and subsystems (both for LRA coordinator and LRA participant respectively) by running CLI commands.
+For your convenience, this quickstart batches the commands into a `enable-microprofile-lra.cli` script provided in the root directory
+of this quickstart.
+
+. Before you begin, make sure you do the following:
+
+* xref:back_up_standalone_server_configuration[Back up the {productName} standalone server configuration] as described above.
+* xref:start_the_eap_standalone_server[Start the {productName} server with the standalone default profile] as described above.
+
+. Review the `enable-microprofile-lra.cli` file in the root of this quickstart directory. It enables two extensions and adds
+two subsystems, one for LRA coordinator and one for LRA participant respectively.
+. Open a new terminal, navigate to the root directory of this quickstart, and run the following command, replacing `__{jbossHomeName}__`
+with the path to your server:
++
+[source,subs="+quotes,attributes+",options="nowrap"]
+----
+$ __{jbossHomeName}__/bin/jboss-cli.sh --connect --file=enable-microprofile-lra.cli
+----
++
+NOTE: For Windows, use the `__{jbossHomeName}__\bin\jboss-cli.bat` script.
++
+
+You should see the following result when you run the script:
++
+[source,options="nowrap"]
+----
+The batch executed successfully
+----
+
+. Stop the {productName} server.
+
+== Review the Modified Server Configuration
+
+After stopping the server, open the `__{jbossHomeName}__/standalone/configuration/{configFileName}` file and review the changes.
+
+. The script added the following two extensions:
++
+[source,xml,options="nowrap"]
+----
+<extension module="org.wildfly.extension.microprofile.lra-coordinator"/>
+<extension module="org.wildfly.extension.microprofile.lra-participant"/>
+----
++
+
+. And also the following two subsystems:
++
+[source,xml,options="nowrap"]
+----
+<subsystem xmlns="urn:wildfly:microprofile-lra-coordinator:1.0"/>
+<subsystem xmlns="urn:wildfly:microprofile-lra-participant:1.0"/>
+----
++
+
+[[solution]]
```

Comment:
It looks now a little bit odd:
![image](https://github.com/wildfly/quickstart/assets/18175164/ca8043c8-7749-4bff-be95-62300a4cd408)

Reply:
that is probably due to the line 121 +